packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -163,9 +163,16 @@ class BondWithdrawalWatcher extends BaseWatcherWithEventHandlers {
         `checking ${dbTransfers.length} unbonded transfers db items`
       )
     }
+
+    const headBlockNumber = await this.bridge.getBlockNumber()
     const promises: Promise<any>[] = []
     for (let dbTransfer of dbTransfers) {
-      const { transferId } = dbTransfer
+      const { transferId, transferSentBlockNumber } = dbTransfer
+      const targetBlockNumber = transferSentBlockNumber + getSafeWaitConfirmations(this.chainSlug)
```

Comment:
alternatively, this

```js
getSafeWaitConfirmations(this.chainSlug)
```

can be read like this

```js
this.bridge.waitConfirmations
```

Reply:
good call. resolved with 5cd3f